### DISCUSSION TITLE
How to add API key for Frigate+ but on Frigate truenas scale app?

### DISCUSSION THREAD

**Discussion by Yonny24** [blakeblackshear/frigate#7516] [Ask A Question]
Just moved my frigate from a proxmox container to truenas scale app.
But I lost access to my frigate+ dashboard and can't seem to update the existing docker image with a new api key.

I can add the variable under manage docker containers but the Save button is not available to press.

Anyone else tried this?

**Comment by novikovaleksei9-glitch** [comment]
+1

**Reply by ElectronicBattle** [reply]
This is what I did:

go into the apps page in the TN webUI and find Frigate and press edit:
<img width="468" height="316" alt="image" src="https://github.com/user-attachments/assets/86dfdc93-2961-4e5a-8c5e-03dd52534924" />

then go here:
<img width="448" height="60" alt="image" src="https://github.com/user-attachments/assets/b989a17a-e16e-40ab-8899-b255240cf232" />

add the API key:
<img width="438" height="240" alt="image" src="https://github.com/user-attachments/assets/c8f140d9-6a31-480a-8c61-c4b792908f2f" />

then save it and once the Frigate web page has loaded, you should have  UI access to Frigate+ 

( I confused myself regarding this matter quite recently: https://github.com/blakeblackshear/frigate/discussions/22434 )
